lib/slack-ruby-bot/rspec/support/slack-ruby-bot/it_behaves_like_a_slack_bot.rb

```diff
@@ -12,4 +15,35 @@
       expect { subject }.to raise_error RuntimeError, "Missing ENV['SLACK_API_TOKEN']."
     end
   end
+
+  context 'when SLACK_API_TOKEN is defined in ENV but not config' do
+    it 'sets config.token from ENV' do
+      allow(ENV).to receive(:[]).and_call_original.at_least(:once)
+      allow(ENV).to receive(:[]).with('SLACK_API_TOKEN').and_return(token)
+      SlackRubyBot.configure { |config| config.token = nil }
```

Comment:
This pattern diverges from the existing, I do feel it makes the tests more isolated from the setup in spec_helper. What do you think @dblock?

Reply:
I like it.